includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -26,9 +26,28 @@ public function setHrefs( $hrefs ) {
 	 * @return string full URL, in case of lang specific URL missing, default one is returned
 	 */
 	public function getHref( $name, $lang ) {
+		$lang = $this->getLangWithFallback( $lang );
+
 		return $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
 	}
 
+	private function getLangWithFallback( $lang ) {
+		if ( isset( $this->hrefs[ $lang ] ) ) {
+			return $lang;
+		}
+
+		$fallbacks = Language::getFallbacksFor( $lang );
+		foreach ( $fallbacks as $fallbackCode ) {
+			// All languages fallback to en, but we use that for English-specific
+			// URLs, so we want to fallback only to default, rather than en
+			if ( $fallbackCode !== 'en' && isset( $this->hrefs[ $fallbackCode ] ) ) {
+				return $fallbackCode;
+			}
+		}
+
+		return 'default';
+	}
+
 	private $hrefs = [
 		'default' => [
 			'fan-communities' => 'http://fandom.wikia.com/explore',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "fan-communities" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)